modules/gitops-release-notes-1-13-0.adoc

```diff
@@ -0,0 +1,119 @@
+
+* With this update, you can perform canary-based Argo Rollouts deployments by using OpenShift Routes-based traffic management. Argo Rollouts supports traffic splitting by modifying the percentage of traffic that reaches the canary or stable application version during a deployment. link:https://issues.redhat.com/browse/GITOPS-2400[GITOPS-2400]
+
+* With this update, the Argo CD Applications in any namespace feature is promoted from Technology Preview (TP) to General Availability (GA). link:https://issues.redhat.com/browse/GITOPS-3796[GITOPS-3796]
+
+* With this update, the following new commands are introduced for the Apps in any namespace feature:
+** `argocd proj add-source-namespace` - Use this command to add source namespaces to `AppProject.Spec.SourceNamespaces`, for whitelisting specific namespaces for application creation.
+** `argocd proj delete-source-namespace` - Use this command to remove source namespaces from `AppProject.Spec.SourceNamespaces`, and for blacklisting specific namespaces. link:https://issues.redhat.com/browse/GITOPS-4045[GITOPS-4045]
+
+* With this update, you can customize permissions for a cluster-scoped Argo CD instance by creating user-defined `ClusterRoles` and `ClusterRoleBindings` for the Argo CD Application Controller and server components. link:https://issues.redhat.com/browse/GITOPS-2614[GITOPS-2614]
+
+* With this update, you can customize the Keycloak route name in the {gitops-title} Operator. This update also enables support for complex routing scenarios, such as dividing routes among various ingress controller shards. link:https://issues.redhat.com/browse/GITOPS-4451[GITOPS-4451]
+
+* With this update, the default TLS termination policy of the Argo CD server's Route changed from `Passthrough` to `Reencrypt`. As a result, the TLS connection now relies on the default ingress operator certificate instead of the self-signed certificate generated by the {gitops-title} Operator. link:https://issues.redhat.com/browse/GITOPS-3918[GITOPS-3918]
+
+* With this update, Argo CD upstream is enhanced to ensure that resource deletion is managed properly when using sync-waves. Now, Argo CD waits for resources to be entirely deleted before advancing to the next sync-wave in the deletion process. This approach ensures a controlled and predictable workflow, enhancing the user experience and preventing potential issues that might arise from not waiting for resources to be fully recycled before proceeding to the next sync-wave. link:https://issues.redhat.com/browse/GITOPS-2642[GITOPS-2642]
+
+* With this update, Argo CD CLI has been enhanced by enabling the `--app-namespace` flag for various application-related commands. This flag allows users to specify the namespace for operations such as waiting, rolling back, patching, editing, diffing, viewing history, syncing, and deleting applications. As a result, the following commands are updated in {gitops-title}: link:https://issues.redhat.com/browse/GITOPS-3834[GITOPS-3834]
```

Comment:
Would suggest moving Jira ticket to after first sentence to avoid breaking up this list.

Reply:
Fixed.